packages/solidity-contracts/scripts/hardhat/depositETH.ts

```diff
@@ -0,0 +1,51 @@
+import { Wallet, parseEther } from 'ethers';
+import type { Signer } from 'ethers';
+import { isB256, isBech32, toB256 } from 'fuels';
+import { task } from 'hardhat/config';
+
+task('depositETH', 'grants a given role to a given adress')
+  .addFlag('env', 'use this flag to send transactions from env var PRIVATE_KEY')
```

Comment:
nit: can add a flag `i` (interactive) to enter private key on prompt for flexibility.